resources/templates/application.properties

```diff
@@ -1,2 +1,9 @@
-autoRenew.targetTypes=
-hedera.config.version=0
+ledger.id=0x01
```

Comment:
Can you please confirm the `ledger.id` param here. Currently `solo` uses ledger_id 298: https://github.com/hashgraph/solo/blob/df04772b846e8af62090ac67e829a8bc56131ed7/src/core/constants.mjs#L37

Reply:
This is same as `0x12a`: https://github.com/hashgraph/hedera-json-rpc-relay?tab=readme-ov-file#configuration